tests/samples_test.py

```diff
@@ -0,0 +1,420 @@
+import pytest
+
+import numpy
+
+import smolyay.samples
```

Comment:
Just import smolyay to make sure the internal package is working as intended
```suggestion
import smolyay
```